torch_em/data/raw_dataset.py

```diff
@@ -107,7 +107,13 @@ def _get_sample(self, index):
                 sample_id += 1
                 if sample_id > self.max_sampling_attempts:
                     raise RuntimeError(f"Could not sample a valid batch in {self.max_sampling_attempts} attempts")
-
+        if self.patch_shape is not None:
+            raw = ensure_patch_shape(
+                raw=raw,
+                labels=None,
+                patch_shape=self.patch_shape,
+                with_channels=self._with_channels
```

Comment:
The argument is called `have_raw_channels`. See also my previous comment.

Reply:
corrected it accordingly